JS/Lesson24/goksel/style.css

```
@@ -0,0 +1,142 @@
+* {
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
+
+body {
+    background-color: #000;
+}
+
+.form-container {
+    background-color: #fff;
+    border-radius: 9px;
+    width: 900px;
+    margin: 75px auto;
+}
+
+/* .form is for to remove content */
+.form {
+    display: none;
+}
+
+/* .active-form is for to make active content of current form */
+.active-form {
+    background-color: #fff;
+    border-radius: 9px;
+    width: 900px;
+    margin: 75px auto;
+    display: block;
+}
+
+#myForm {
+    display: flex;
+}
+
+/* codes for to edit main form container*/
+.left-container {
+    padding: 35px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.left-container>h1 {
+    text-align: center;
+    font-weight: 800;
+    font-size: 44px;
+}
+
+.left-container>ul {
+    margin-bottom: 45px;
+}
+
+.left-container>ul>li {
+    list-style-type: none;
+    font-weight: 300;
+    margin-left: -45;
+    line-height: 2.5;
+}
+
+.left-container>ul>li>i {
+    color: #FF6253;
+    font-size: 20px;
+}
+
+label:first-of-type {
+    font-size: 12px;
+    font-weight: 700;
+    display: inline-block;
+}
+
+#email {
+    display: block;
+    width: 100%;
+    border-radius: 5px;
+    border: 1px solid #D3D3D3;
+    padding: 15px;
+    margin-top: 10px;
+}
+
+.submit-btn {
+    width: 100%;
+    color: #fff;
+    background-color: #242742;
+    border: none;
+    border-radius: 5px;
+    padding: 15px;
+    margin-top: 25px;
+    cursor: pointer;
+}
+
+/* codes for to edit img */
+.right-container {
+    padding: 25px;
+}
+
+.label-error {
+    display: flex;
+    justify-content: space-between;
+}
+
+#email-error {
+    color: #FF6253;
+    font-size: 12px;
+}
+
+/* Codes for to edit Second form; after javascript worked and first form submitted */
```

Normally you should not leave comments like this, when things are named well, it should be strait forward to understand what code is for what just by reading it or searching for a specific class name.
But if you really want to keep this comment, here's fixed version:
```suggestion
/* Code for success message dialog. */
```

Don't say "codes".  Code is a plural form already when used in programming.